include/CL/sycl/device.hpp

```diff
@@ -210,11 +210,9 @@ class device
       \todo
   */
   template <info::device Param>
-  inline auto get_info() const {
+  auto get_info() const {
```

Comment:
I think that the changes from here will break the ParallelSTL compiled with triSYCL from https://github.com/KhronosGroup/SyclParallelSTL even if it is not yet in the current non-regression tests.
Perhaps putting this in some "#ifndef _WIN32"?
To be looked at by @JoanThibault.